Add Checkmarx scan results to Port (#439)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/packages/checkmarx.md

---
sidebar_position: 9
description: Ingest Checkmarx KICS scan into your catalog
---

import PythonScript from './resources/checkmarx/\_example_python_script.mdx'
import CheckmarxBlueprint from './resources/checkmarx/\_example_checkmarx_blueprint.mdx'
import CheckmarxWebhookConfig from './resources/checkmarx/\_example_checkmarx_webhook_config.mdx'

# Checkmarx KICS

> In this example you are going to create a `checkmarxScan` blueprint that ingests all scan results in your Checkmarx KICS file using a combination of Port's [API](../../../api/api.md) and [webhook functionality](../../webhook.md).

Reviewer: ```suggestion
In this example, you will create a `checkmarxScan` blueprint that ingests all scan results in your Checkmarx KICS file using a combination of Port's [API](../../../api/api.md) and [webhook functionality](../../webhook.md).
```

> To ingest the scan results to Port, a script that sends information about the scans according to the webhook configuration is used.

Reviewer: ```suggestion
To ingest the scan results to Port, use a script that sends information about the scans according to the webhook configuration.
```
